### ISSUE TITLE
Arithmetic Operations in Projections Produce Zero Results

### ISSUE THREAD

**Issue by apavlo** [cmu-db/terrier#714]
If we modify a column in the output clause of a SELECT statement, the result has zero rows in it:

## How To Reproduce

1. Create a table
    ```
    terrier=# CREATE TABLE xxx (id INT);
    CREATE TABLE
    ```
2. Load the table
    ```
    terrier=# INSERT INTO xxx VALUES (1),(2),(3);
    INSERT 0 3
    terrier=# SELECT * FROM xxx;
     id 
    ----
      1
      2
      3
    (3 rows)
    ```
3. Query the table by adding 1 to the id column:
    ```
    # SELECT id + 1 FROM xxx;
     + id + INTEGER 
    ----------------
    (0 rows)
    ```

**Comment by harsh141994** [comment]
Will be solved with the merge of pr # 705.

**Comment by lmwnshn** [comment]
Was fixed by #705.
